en/mozillavpn.xlf

```diff
@@ -0,0 +1,505 @@
+  </body></file>
+  <file datatype="plaintext" original="../src/states/StateAuthenticating.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="waitForSignIn">
+      <source xml:space="preserve">Waiting for sign in and subscription confirmation&#8230;</source>
+      <context-group purpose="location"><context context-type="linenumber">15</context></context-group>
+    </trans-unit>
+    <trans-unit id="cancelAndRetry">
+      <source xml:space="preserve">Cancel and try again</source>
+      <context-group purpose="location"><context context-type="linenumber">43</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/states/StatePostAuthentication.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="quickAccess">
+      <source xml:space="preserve">Quick access</source>
+      <context-group purpose="location"><context context-type="linenumber">21</context></context-group>
+    </trans-unit>
+    <trans-unit id="statusBarIntro">
+      <source xml:space="preserve">You can quickly access Mozilla VPN from your status bar.</source>
+      <context-group purpose="location"><context context-type="linenumber">29</context></context-group>
+    </trans-unit>
+    <trans-unit id="wip">
+      <source xml:space="preserve">WIP</source>
+      <context-group purpose="location"><context context-type="linenumber">62</context></context-group>
+    </trans-unit>
+    <trans-unit id="continue">
+      <source xml:space="preserve">Continue</source>
+      <context-group purpose="location"><context context-type="linenumber">95</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/states/StateSubscriptionNeeded.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="subscriptionNeeded">
+      <source xml:space="preserve">Subscription needed</source>
+      <context-group purpose="location"><context context-type="linenumber">17</context></context-group>
+    </trans-unit>
+    <trans-unit id="subscribeNow">
+      <source xml:space="preserve">Subscribe now</source>
+      <context-group purpose="location"><context context-type="linenumber">23</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/views/ViewDevices.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="currentDevice">
+      <source xml:space="preserve">Current Device</source>
+      <context-group purpose="location"><context context-type="linenumber">112</context></context-group>
+    </trans-unit>
+    <trans-unit id="addedltHour">
+      <source xml:space="preserve">Added less than a hour ago</source>
+      <context-group purpose="location"><context context-type="linenumber">120</context></context-group>
+    </trans-unit>
+    <trans-unit id="addedXhoursAgo">
+      <source xml:space="preserve">Added %1 hours ago</source>
```

Comment:
This is going to be challenging, because we don't support plurals in XLIFF :-(

Reply:
We still don't know the correct messages to show here. Can we ignore it for now?

Reply:
Yes. I think the only solution would be to actually show a time, since we can't support plurals, e.g. `Added a few hours ago (TIME)`.

Do you already have a sense of how much of this content is final, i.e. reviewed from UX/copy?